lighthouse-core/report/report-generator.js

```diff
@@ -222,7 +222,11 @@ class ReportGenerator {
     if (reportContext === 'devtools') {
       return [];
     } else {
-      return [fs.readFileSync(path.join(__dirname, './scripts/lighthouse-report.js'), 'utf8')];
+      return [
+        fs.readFileSync(path.join(__dirname, './scripts/logger.js'), 'utf8'),
```

Comment:
nit: can drop the `path.join` and just `fs.readFileSync(__dirname + '/scripts/logger.js'), 'utf8')`